### PR TITLE
Add search box to projects page.

### PR DESCRIPTION
**Pull request by Gaelan**
## Problem

It'd be nice to have an easy way to filter for projects matching a given keyword.

## Solution

Implement a search box with a bit of a JS.

Just plain JS, no frameworks, no jQuery - in all my years of web dev, I'm not sure I've ever done that! A little clunkier, maybe, but it feels nice.

UI-wise, I don't use the USWDS search component, as that assumes a search button; we don't need one here, we can filter as the user types. Instead, I did something custom (or, rather, using more basic USWDS components) to put a search con on the left side of the field, along with an old-fashioned label:

screenshot of the projects page, now with a search box (as described). The search box has a label (with the text 'filter') and a little magnifying glass icon inside the field.

## Test Plan

Tested manually.

### CODE REVIEW

File: site/projects.liquid [hunk 1]
```diff
@@ -1,11 +1,18 @@
 ---
 layout: base
 ---
+<script type="module" src="/assets/js/projects.js"></script>
 <div class="grid-container">
-  The Centers for Medicare and Medicaid Services develops or funds many open-source projects. Here are some of them:
+  <p>
+    The Centers for Medicare and Medicaid Services develops or funds many open-source projects. Here are some of them:
```
Comment (decause-gov) [inline site/projects.liquid:7]: s/funds/supports

File: src/js/projects.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const filterBox = document.getElementById("filter-input")
+filterBox.addEventListener("input", () => {
+  const query = filterBox.value.toLowerCase()
+  document.querySelectorAll(".project-card").forEach((card) => {
+    card.hidden = !(
+      query == "" || card.textContent.toLowerCase().includes(query)
```
Comment (decause-gov) [inline src/js/projects.js:6]: +1 toLower+includes queries 

LGTM

**Review by decause-gov — APPROVED** [review]

File: site/projects.liquid (hunk 1 above)
Comment (Gaelan) [reply site/projects.liquid:7]: Sure, done.

That whole paragraph was mostly written as a placeholder anyway, happy to replace with something better if it comes along. Probably good enough for now.

**Comment by Gaelan** [comment]
(Force pushed some HTML changes - meant to do that Friday, oops. Just adds a little search icon to the search box.)

**Pull request merged**